src/components/dialog/ImportModelTemplateDialog.vue

```diff
@@ -11,10 +11,20 @@
       {{ $t(`actions.models.import.dialog.name`) }}
     </template>
     <template v-slot:default>
-      <div class="q-pb-md text-subtitle2">
-        {{ $t('page.models.template.selectedTemplate', { template: modelTemplate?.type }) }}
+      <div class="q-pb-md text-center text-subtitle2">
+        {{ modelTemplate?.type }}
```

Comment:
display "undefined" if modelTemplate does not exist ?

Reply:
I don't think `modelTemplate` can _not exist._ In any case, if it does not exist, it is not possible to open the dialog

Reply:
add `v-if="modelTemplate"` and remove optional chaining operator